kratos/includes/model_part.h

```diff
@@ -1808,11 +1808,6 @@ class KRATOS_API(KRATOS_CORE) ModelPart final
         return mMeshes;
     }
 
-    std::string& Name()
-    {
-        return mName;
-    }
-
```

Comment:
yikes, time to go :wave:

Reply:
In this particualr case yes, it makes sense

Reply:
~~~
add_model_part_to_python.cpp:46:25: error: no match for 'operator=' (operand types are 'const std::string' {aka 'const std::__cxx11::basic_string<char>'} and 'const std::string' {aka 'const std::__cxx11::basic_string<char>'})
~~~

Looks like python binding is a bit hardcoded

Reply:
Algo agree with this one